### ISSUE TITLE
collecting steps for other convenient ucm deployment methods

### ISSUE THREAD

**Issue by aryairani** [unisonweb/unison#1475]
These would list any package dependencies such as libtinfo5 that our version of haskeline wants.

### Ubuntu PPA:
https://askubuntu.com/a/71516

### Debian repo:
https://www.jfrog.com/confluence/display/BT/Debian+Repositories
 / bug @stew 😃 

### Redhat/yum repo:
https://www.jfrog.com/confluence/display/BT/YUM+Repositories

### Misc
To have Azure perform signing with unisoncomputing keys, we can use [Secure Files](https://docs.microsoft.com/en-us/azure/devops/pipelines/library/secure-files?view=azure-devops).

**Comment by stew** [comment]
I created a repo for creating debian packages for ucm, and cut a release:

https://github.com/stew/unison-debian/releases/tag/0.0.1_M1l

I'm at least producing `.deb`s now. I need to further document how to build them and how to build them for different debian/ubuntu versions and then work on scripts to package the resulting .debs into a debian repo that people can apt-get from.

**Comment by aryairani** [comment]
@stew NEAT.  It's more involved than I realized!  Made me happy to see the binary dependencies listed though.

**Comment by stew** [comment]
Yeah, this is more involved because I'm building it from source against the debian library, and I've kept the package policy compliant so theoretically this could be uploaded mostly as is to debian / ubuntu proper some day. This could be packaged much more simply if all one wanted to do was to copy the already compiled binary from the unison release

**Comment by ceedubs** [comment]
There's also a Nix derivation for `ucm` in nixpkgs [here](https://github.com/NixOS/nixpkgs/blob/master/pkgs/development/compilers/unison/default.nix). So anyone using Nix can refer to the `unison-ucm` package.

In addition to being a deployment option, for those who use Nix this makes it really easy to drop into a shell and try out `ucm` without polluting their global `PATH`:

```
❯ nix-shell -p unison-ucm --command ucm

   _____     _             
  |  |  |___|_|___ ___ ___ 
  |  |  |   | |_ -| . |   |
  |_____|_|_|_|___|___|_|_|
  
  Welcome to Unison!
  
  I'm currently watching for changes to .u files under ~
  
  Type help to get help. 😎

.>
```
